.github/workflows/benchmark.yml

```diff
@@ -21,5 +25,12 @@ jobs:
         run: pip install -r requirements.txt
       - name: Run Benchmarks
         run: |-
-          asv machine --yes
-          asv run
+          asv machine --machine ${{ matrix.os }} --yes
+          asv run "HEAD -n 10" --skip-existing
```

Comment:
I think we can run it for just a single commit each day. 

I think at some point we can then change it to run only once a week. (But let's leave it at daily while we're doing lots of work here)